tests/safeds/data/tabular/containers/_table/test_get_similar_columns.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.exceptions._data import UnknownColumnNameError
+
+
+@pytest.mark.parametrize(
+    ("table", "column_name", "expected"),
+    [
+        (Table({"column1": ["col1_1"], "x": ["y"], "cilumn2": ["cil2_1"]}), "col1", ["column1"]),
+        (
+            Table(
+                {
+                    "column1": ["col1_1"],
+                    "col2": ["col2_1"],
+                    "col3": ["col2_1"],
+                    "col4": ["col2_1"],
+                    "cilumn2": ["cil2_1"],
+                },
+            ),
+            "clumn1",
+            ["column1", "cilumn2"],
+        ),
+        (
+            Table({"column1": ["col1_1"], "x": ["y"], "cilumn2": ["cil2_1"]}),
+            "notexisting",
+            ["x"],
+        ),
```

Comment:
This doesn't match the ID. The ID says `no similar`.

```suggestion
               (
            Table({"column1": ["a"], "column2": ["b"], "column3": ["c"]}),
            "notexisting",
            [],
        ),
```